classwrap.d.ts

```diff
@@ -0,0 +1,17 @@
+export as namespace classwrap
+
+export interface ClassSet {
+  [key: string]: boolean | any | ClassSet
```

Comment:
Here `any` is sufficient to override all types.
But I took the habit to type the default or most awaited type.

Because here `any` it used to peform truthy/falsy check. We are waiting at the end a `boolean`.